### DISCUSSION TITLE
Vulnerability of dependency "go.etcd.io/etcd, github.com/blevesearch/bleve"

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [dgraph-io/dgraph#8922] [Ideas]
Hello, we are a team researching the dependency management mechanism of Golang. During our analysis, we came across your project and noticed that it contains a vulnerability (CVE-2023-32082, CVE-2020-15113, CVE-2020-15112, CVE-2020-15106, CVE-2022-31022). 
In your project, the go.etcd.io/etcd package is being used at version go.etcd.io/etcd v0.5.0-alpha.5.0.20190108173120-83c051b701d3, but the patched version is v3.5.9. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v3.5.9 or higher.
the github.com/blevesearch/bleve package is being used at version github.com/blevesearch/bleve v1.0.13, but the patched version is v2.3.3. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v2.3.3 or higher.
Thank you for your attention to this matter.

**Comment by matthewmcneely** [comment]
etcd-raft is now at 3.5.23 in our go.mod
